store/db/sqlite/nest.go

```diff
@@ -0,0 +1,155 @@
+package sqlite
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/usememos/memos/store"
+)
+
+func (d *DB) CreateNest(ctx context.Context, create *store.Nest) (*store.Nest, error) {
+	fields := []string{"`uid`", "`creator_id`"}
+	placeholder := []string{"?", "?"}
+	args := []any{create.UID, create.CreatorID}
+
+	stmt := "INSERT INTO `nest` (" + strings.Join(fields, ", ") + ") VALUES (" + strings.Join(placeholder, ", ") + ") RETURNING `id`, `created_ts`, `updated_ts`, `row_status`"
+	if err := d.db.QueryRowContext(ctx, stmt, args...).Scan(
+		&create.ID,
+		&create.CreatedTs,
+		&create.UpdatedTs,
+		&create.RowStatus,
+	); err != nil {
+		return nil, err
+	}
+
+	return create, nil
+}
+
+func (d *DB) ListNests(ctx context.Context, find *store.FindNest) ([]*store.Nest, error) {
+	where, args := []string{"1 = 1"}, []any{}
+
+	if v := find.ID; v != nil {
+		where, args = append(where, "`nest`.`id` = ?"), append(args, *v)
+	}
+	if v := find.UID; v != nil {
+		where, args = append(where, "`nest`.`uid` = ?"), append(args, *v)
+	}
+	if v := find.CreatorID; v != nil {
+		where, args = append(where, "`nest`.`creator_id` = ?"), append(args, *v)
+	}
+	if v := find.RowStatus; v != nil {
+		where, args = append(where, "`nest`.`row_status` = ?"), append(args, *v)
+	}
+	if v := find.CreatedTsBefore; v != nil {
+		where, args = append(where, "`nest`.`created_ts` < ?"), append(args, *v)
+	}
+	if v := find.CreatedTsAfter; v != nil {
+		where, args = append(where, "`nest`.`created_ts` > ?"), append(args, *v)
+	}
+	if v := find.UpdatedTsBefore; v != nil {
+		where, args = append(where, "`nest`.`updated_ts` < ?"), append(args, *v)
+	}
+	if v := find.UpdatedTsAfter; v != nil {
+		where, args = append(where, "`nest`.`updated_ts` > ?"), append(args, *v)
+	}
```

Comment:
This functionality is probably not something we will need for a nest

Reply:
Agree, removed.